src/python_be.cc

```diff
@@ -387,10 +387,21 @@ ModelInstanceState::SaveRequestsToSharedMemory(
     auto err = TRITONBACKEND_RequestTrace(request, &triton_trace);
     if (err != nullptr) {
       triton_trace = nullptr;
+      LOG_MESSAGE(TRITONSERVER_LOG_ERROR, TRITONSERVER_ErrorMessage(err));
```

Comment:
Wasn't this error intentionally silenced as described by comment above?

Reply:
This one should just print it, not fail a response. The comment above explains why we don't do:
```
RETURN_IF_ERROR(TRITONBACKEND_RequestTrace(request, &triton_trace));
```
here. I think at least printing error message should be helpful with the debug.

Reply:
Actually, I'll remove it. Otherwise when Triton is compiled without tracing, it'll keep logging message.

Reply:
Yeah that's what I was worried about

Reply:
Good catch!

Reply:
updated